sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -430,40 +430,40 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 [coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
 [coc_contact]: mailto:opencode@microsoft.com
 [create_new_resource]: https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account?tabs=multiservice%2Cwindows#create-a-new-azure-cognitive-services-resource
-[differentiate_custom_forms_with_labeled_and_unlabeled_models]: src/samples/java/com/azure/ai/formrecognizer/AdvancedDiffLabeledUnlabeledData.java
+[differentiate_custom_forms_with_labeled_and_unlabeled_models]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer//AdvancedDiffLabeledUnlabeledData.java
```

Comment:
consider removing extra `/` in `formrecognizer//AdvancedDiffLabeledUnlabeledData.java`